Library/Homebrew/build_options.rb

```diff
@@ -101,6 +101,15 @@ def unused_options
     @options - @args
   end
 
+  # @private
+  def invalid_options
+    (@options | @args) - @options
```

Comment:
No, `#unused options` lists valid options for the formula not in ARGV. `#invalid_options` lists ARGV 'options' that are not listed in the formula's options at all. See the tests for an example :)